### PR TITLE
Fix #13 - Tests for validator w rand strings

### LINKED ISSUE #13: Write tests for helpers/validator

### PR DESCRIPTION
**Pull request by rignatus**
Created a function to generate random strings in order to make the tests more effective.

### CODE REVIEW

File: helpers/validator/validator_test.go [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+				"For", pair.value,
+				"expected result", pair.result,
+				"got", res,
+			)
+		}
+	}
+}
+
+// new source for rand func, seeding it with time.Now().UnixNano()
+// for different outputs at differents runs
```
Comment (andreimuntean) [inline helpers/validator/validator_test.go]: Nit: differents.

File: helpers/validator/validator_test.go [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+	"testing"
+	"time"
+)
+
+type testPair struct {
+	value  string
+	result bool
+}
+
+var testsEmail = []testPair{
```
Comment (andreimuntean) [inline helpers/validator/validator_test.go]: Nit: I think _emailTests_, _messageTests_ and _nameTests_ might be easier to read.

File: helpers/validator/validator_test.go [hunk 3]
```diff
@@ -0,0 +1,111 @@
+package validator
+
+import (
+	"math/rand"
+	"strings"
+	"testing"
+	"time"
+)
+
+type testPair struct {
```
Comment (andreimuntean) [inline helpers/validator/validator_test.go]: What do you think sounds better: testPair or testCase? It's fine either way.

File: helpers/validator/validator_test.go [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+	{randString(29), true},
+	{randString(1), true},
+	{randString(1) + strings.Repeat(" ", 29), true},
+	{randString(1) + strings.Repeat(" ", 30), false},
+	{randString(31), false},
+	{strings.Repeat(" ", int(src.Int63()%31)), false},
+}
+
+func TestIsValidEmail(t *testing.T) {
+	for _, pair := range testsEmail {
```
Comment (andreimuntean) [inline helpers/validator/validator_test.go]: If you rename _testsEmail_ to _emailTests_, you could rename _pair_ to _emailTest_ (same below).

**Review by andreimuntean — COMMENTED** [review]
Mentioned a few nits. Looks great!

File: helpers/validator/validator_test.go (hunk 3 above)
Comment (rignatus) [reply helpers/validator/validator_test.go]: testCase. Will change accordingly.

File: helpers/validator/validator_test.go (hunk 2 above)
Comment (rignatus) [reply helpers/validator/validator_test.go]: Indeed. Will change accordingly.

File: helpers/validator/validator_test.go (hunk 4 above)
Comment (rignatus) [reply helpers/validator/validator_test.go]: Will do.

**Pull request merged**